src/java/us/kbase/mobu/shell_launcher/ShellLauncher.java

```diff
@@ -0,0 +1,48 @@
+package us.kbase.mobu.shell_launcher;
+
+import java.io.File;
+import java.io.IOException;
+import java.lang.InterruptedException;
+import java.lang.ProcessBuilder;
+import java.lang.Process;
+import java.util.Map;
+import java.nio.file.Paths;
+
+import org.yaml.snakeyaml.Yaml;
+
+import us.kbase.mobu.util.DirUtils;
+import us.kbase.mobu.util.TextUtils;
+import us.kbase.mobu.ModuleBuilder;
+
+public class ShellLauncher {
+    File moduleDir;
+    String moduleName;
+    String workDir;
+
+    public ShellLauncher() throws IOException {
+        moduleDir = DirUtils.findModuleDir();
+        String kbaseYml = TextUtils.readFileText(new File(moduleDir, "kbase.yml"));
+        @SuppressWarnings("unchecked")
+        Map<String,Object> config = (Map<String, Object>)new Yaml().load(kbaseYml);
+        moduleName = (String)config.get("module-name");
+        workDir = Paths.get(moduleDir.toPath().toString(), "test_local", "workdir").toString();
+    }
+
+    public Process exec() throws IOException, InterruptedException {
+        // This command looks like:
+        // `docker run -i -t -v WORKDIR:/kb/module/work test/module:latest bash`
```

Comment:
Are there long forms of the flags? Would be easier to read if so. If not, document what the flags mean